### PR TITLE
Cistup ec

### PR DESCRIPTION
**Pull request by deepalics0044**
Mode and purpose changes

### CODE REVIEW

File: www/css/style.css [hunk 1]
```diff
@@ -905,7 +905,11 @@ button.button.back-button.buttons.button-clear.header-item {
         font-size: 9px;
         position: absolute;
         left: 1%;
+<<<<<<< HEAD
         bottom:15px;
+=======
```
Comment (shankari) [inline www/css/style.css]: merge conflict

File: www/js/diary/list.js [hunk 2]
```diff
@@ -1,25 +1,42 @@
...
   var MODE_CONFIRM_KEY = "manual/mode_confirm";
   var PURPOSE_CONFIRM_KEY = "manual/purpose_confirm";
   var QUICKRIDE_CONFIRM_KEY = "manual/quickride_confirm";
 	 var PARKING_CONFIRM_KEY = "manual/parking_confirm";
+=======
+    Config, PostTripManualMarker, ConfirmHelper, nzTour, KVStore, Logger, UnifiedDataLoader, $ionicPopover) {
+  console.log("controller DiaryListCtrl called");
+    var MODE_CONFIRM_KEY = "manual/mode_confirm";
+    var PURPOSE_CONFIRM_KEY = "manual/purpose_confirm";
+>>>>>>> 9fc4bfa48a421d8e10e5f38a847654206cac3000
```
Comment (shankari) [inline www/js/diary/list.js]: merge conflict

**Comment by shankari** [comment]
@deepalics0044 to test the PR before submission, I would suggest using the devapp so that you can also livereload the changes. Building the apk works too, of course.

**Comment by shankari** [comment]
Also, I just merged #553 to master, so you should be able to pull directly from master as well.
Will probably have merge conflicts again, though...

**Comment by shankari** [comment]
@deepalics0044 If you are not familiar with resolving merge conflicts, you may find it easier to create a new branch from master and copy over the changes you made to cistup. They are likely to be smaller (icons, modes) and more amenable to the "copying over files" style of merging

**Comment by deepalics0044** [comment]
The branch master is quite different from cistup-ec. I can't really figure
out how my changes into master(a new branch created from master) will give
it a familiar look.

**Comment by deepalics0044** [comment]
Are there only two merge conflicts or you just reviewed two.

**Comment by shankari** [comment]
If you want to stick to the "copying over files" style of merging", you can probably just copy the files that were changed in #553 
AFAIK, there were no substantial cci-berkeley changes other than the mode and purpose changes (which are subsumed by this PR) and the cci-about page, which was not modified by #553 

You should definitely test before submitting the next PR though

**Comment by deepalics0044** [comment]
I am unable to solve conflicts there are lots of them.

**Comment by shankari** [comment]
@deepalics0044 It should be fairly mechanical to resolve the conflicts - you almost always want to take the option other than HEAD.
And I have also give you two options for the "copy over" style of merge that you prefer rather than a proper merge conflict resolve.

Have you tried those?

**Comment by deepalics0044** [comment]
I made a new branch cistup-el out of updated master and have made the changes. Testing apk is still left. If succesful,  do I need to push the changes to cistup-ec from cistup -el?

**Comment by shankari** [comment]
@deepalics044 you can submit a PR from any branch to any other branch. Please see github
 for details on how to do so

**Comment by shankari** [comment]
Closing this since cistup has switched to building their own apk

**Pull request closed without merging**